sv_pipeline/genome/load_data_tests.py

```diff
@@ -0,0 +1,295 @@
+                     xstart=1000010000, xstop=1000010000,
+                     sortedTranscriptConsequences=[hl.struct(gene_symbol='OR4F5', gene_id='ENSG00000284662.1_2',
+                                                             predicted_consequence='NEAREST_TSS')],
+                     transcriptConsequenceTerms=['DUP'], svTypeDetail=hl.null('str'), cpxIntervals=NULL_INTERVALS,
+                     geneIds=NULL_STR_ARRAY, filters=['LOW_CALL_RATE'],
+                     samples_num_alt_0=['SAMPLE-5'],
+                     samples_num_alt_1=['SAMPLE-1', 'SAMPLE-3'], samples_num_alt_2=NULL_STR_ARRAY,
+                     genotypes=[hl.struct(sample_id='SAMPLE-1', gq=999, num_alt=1, cn=3),
+                                hl.struct(sample_id='SAMPLE-3', gq=19, num_alt=1, cn=3),
+                                hl.struct(sample_id='SAMPLE-5', gq=31, num_alt=0, cn=2)])
+VARIANT4 = hl.struct(variantId='INS_chr1_10', contig='1', sc=11, sf=0.007703, sn=1428, svType='INS', start=1643228,
+                     end=1643309, sv_callset_Hemi=11, sv_callset_Hom=0, gnomad_svs_ID='gnomAD-SV_v2.1_INS_1_47',
+                     gnomad_svs_AF=0.00130899995565414, pos=1643228, xpos=1001643228,
+                     xstart=1001643228, xstop=1001643228, sortedTranscriptConsequences=[
+        hl.struct(gene_symbol='CDK11B', gene_id='ENSG00000284664', predicted_consequence='INTRONIC')],
+                     transcriptConsequenceTerms=['INS'], svTypeDetail='ME:SVA', cpxIntervals=NULL_INTERVALS,
+                     geneIds=['CDK11B'], filters=NULL_STR_ARRAY,
+                     samples_num_alt_0=['SAMPLE-1', 'SAMPLE-3'],
+                     samples_num_alt_1=['SAMPLE-5'],
+                     samples_num_alt_2=NULL_STR_ARRAY,
+                     genotypes=[hl.struct(sample_id='SAMPLE-1', gq=999, num_alt=0, cn=hl.null('int')),
+                                hl.struct(sample_id='SAMPLE-3', gq=999, num_alt=0, cn=hl.null('int')),
+                                hl.struct(sample_id='SAMPLE-5', gq=1, num_alt=1, cn=hl.null('int')),])
+
+TEST_PASSWORD = 'ExamplePasswd'
+TEST_MT_PATH = 'test_mt/mts'
+TEST_GENCODE_PATH = 'test_gtf/gtf'
+TEST_HOST = 'TEST_HOST'
+TEST_PORT = '9500'
+TEST_GENCODE_RELEASE = 31
+TEST_BLOCK_SIZE = 3000
+TEST_NUM_SHARDS = 10
+
+
+class LoadDataTest(unittest.TestCase):
+    def setUp(self):
+        self.vcf_file = tempfile.mkstemp(suffix='.vcf')[1]
+        with open(self.vcf_file, 'w') as f:
+            f.writelines('\n'.join(VCF_DATA))
+        hl.init()
+        self.mt = hl.import_vcf(self.vcf_file, reference_genome='GRCh38')
+
+    def tearDown(self):
+        hl.stop()
+        os.remove(self.vcf_file)
+
+    @mock.patch('sv_pipeline.genome.load_data.get_sample_subset')
+    @mock.patch('sv_pipeline.genome.load_data.logger')
+    @mock.patch('sv_pipeline.genome.utils.mapping_gene_ids.gene_id_mapping', hl.literal(GENE_ID_MAPPING))
+    def test_sub_and_annotation(self, mock_logger, mock_get_sample):
```

Comment:
This test file is really confusing and hard to read. You basically test a couple functions together here and then test that a bunch of mocks are called in a big, hard to read block. I think the issue is that you should actually break up the main function so it only makes 2 or 3 function calls. The actually test those functions, and for the main test you only need to mock like the 3 functions that exist to test the arg parsing. My recommendations for functions are something like `load_data`, `annotate_data`, and `export_data` but its really up to you